konig-core/src/main/java/io/konig/core/vocab/Konig.java

```diff
@@ -67,7 +67,8 @@
 	// Generic Data Modeling
 
 	public static final URI id = new URIImpl("http://www.konig.io/ns/core/id");
-	public static final URI mediaTypeBaseName = new URIImpl("http://www.konig.io/ns/core/mediaTypeBaseName");
+	public static final URI mediaTypeBaseName = new URIImpl("http://www.konig.io/ns/core/mediaTypeBaseName");
+	public static final URI shapeType = new URIImpl("http://www.konig.io/ns/core/shapeType");

```

Comment:
We should not mint the predicate `konig:shapeType`. Use the standard predicate `rdf:type`.